examples/vision/deeplabv3_plus.py

```diff
@@ -0,0 +1,298 @@
+"""
+Title: Multiclass Semantic Segmentation using DeepLabV3+
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/08/31
+Last modified: 2021/09/1
+Description: Implement DeepLabV3+ architecture for Multi-class Semantic Segmentation
+"""
+"""
+## Introduction
+"""
+
+"""
+Semantic segmentation with the goal to assign semantic labels to every pixel in an image
+is one of the fundamental topics in computer vision. Deep convolutional neural networks
+based on the Fully Convolutional Neural Network show striking improvement over systems
+relying on hand-crafted features on benchmark tasks. In this example, we would implement
+the **DeepLabV3+** model as proposed by the paper [Encoder-Decoder with Atrous Separable
+Convolution for Semantic Image Segmentation](https://arxiv.org/pdf/1802.02611.pdf) for
+multi-class semantic segmentation.
+"""
+
+"""
+## Download Dataset
+"""
+
+import os
+import cv2
+import numpy as np
+from glob import glob
+import tensorflow as tf
+from scipy.io import loadmat
+import matplotlib.pyplot as plt
+
+"""shell
+!gdown https://drive.google.com/uc?id=1B9A9UCJYMwTL4oBEo4RZfbMZMaZhKJaz
+!unzip -q instance-level-human-parsing.zip
```

Comment:
Remove the `!`, it gets added automatically

Reply:
Removed `!`